Frontend/src/app/views/dashboard/project-details/project-details.component.html

```diff
@@ -35,9 +35,19 @@ <h5>
             <input *ngIf="pageMode == 'edit'" type="text" status="primary" placeholder="Tags" nbTagInput (tagAdd)="onTagAdd($event)" fullWidth>
         </nb-tag-list>
     
-        <p>
+        <p *ngIf="pageMode == 'view'">
             <span>{{ details.description }}</span>
         </p>
+
+        <input 
```

Comment:
Since the description does not extend the full page I feel like this should be a textarea that kinda mimics the same look as the text is displayed in before the edit mode.

Reply:
replaced